packages/langium/test/lsp/find-references.test.ts

```diff
@@ -0,0 +1,132 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { createLangiumGrammarServices } from '../../src';
+import { expectFindReferences } from '../../src/test';
+import { expectFunction } from '../fixture';
+
+const text = `
+grammar test hidden(WS)
+
+entry Model: value=RuleA;
+
+interface <|>A {
+    na<|>me: string
+}
+
+interface B extends A {
+    assignmentB: string
+}
+
+type C = A | B;
+
+RuleA returns A: na<|>me=ID;
+
+RuleB returns B: na<|>me=ID assignmentB=ID;
```

Comment:
@Lotes 
I think there is at least an example for `<|>A`, however I would say it would be better to test the `findLeafNodeAtOffset(node: CstNode, offset: number): LeafCstNode | undefined `  in a separate test file if it not already done. https://github.com/langium/langium/blob/02276b565f66a053fe299b8602c08ab761504d14/packages/langium/src/utils/cst-util.ts#L83

Reply:
Yes, actually this is a reason for a separate PR. This PR assumes that the underlying function is already tested and correct.

Reply:
The function is already tested, resolved